site/content/docs/latest/tutorials/managing-package-repositories.md

```diff
@@ -0,0 +1,262 @@
+
+- [Helm Charts](https://helm.sh)
+- [Helm chart releases via Fluxv2](https://fluxcd.io/docs/guides/helmreleases/)
+- [Carvel Packages](https://carvel.dev/kapp-controller/docs/latest/packaging/#package)
+
+To configure the Bitnami Application Catalog, select **Helm Charts**.
+
+**NOTE**: The support for each packaging format is enabled in Kubeapps by means of a plugin. Kubeapps UI shows as selectable only those packaging formats configured in Kubeapps.
+
+> **TIP**: Please refer to the [getting started documentation](./getting-started.md) for more information on how to install Kubeapps and pass custom configuration values.
+
+### Scope: Namespace/Global
+
+Kubeapps allows creating Package Repositories that are available:
+
+- **Namespaced**: in specific namespaces, aligned with the Kubernetes RBAC model where an account can have roles in specific namespaces.
+- **Global**: available cluster-wide to all Kubeapps users.
+
+To configure the Bitnami Application Catalog as a cluster-wide repository, select **Global**.
+
+A Kubeapps Package Repository can be created by anyone with the required RBAC for that namespace. If you have cluster-wide RBAC for creating Package Repositories, you can still create an Package Repository whose packages are available to users in all namespaces by selecting `Global` when creating the repository.
+
+To give a specific user `USERNAME` the ability to create Package Repositories in a specific namespace named `custom-namespace`, grant them both read and write RBAC for `AppRepositories` in that namespace:
+
+```bash
+kubectl -n custom-namespace create rolebinding username-apprepositories-read --user $USERNAME --clusterrole kubeapps:$KUBEAPPS_NAMESPACE:apprepositories-read
+kubectl -n custom-namespace create rolebinding username-apprepositories-write --user $USERNAME --clusterrole kubeapps:$KUBEAPPS_NAMESPACE:apprepositories-write
+```
+
+or to allow other users the ability to deploy packages from Package Repositories in a specific namespace, grant the read access only.
+
+### Storage type
+
+Once selected the packaging format and scope, it is time to set the **storage type** of the repo. Storages available differs for the different packaging formats:
+
+- For **Helm Charts** (both directly with Helm, or via Flux):
+  - `Helm Repository`: a Helm chart repository is an HTTP server that houses an `index.yaml` file and optionally some packaged charts.
+  - `OCI Registry`: The [Open Container Initiative](https://opencontainers.org/) (OCI) defines the specifications and standards for container technologies. This includes the API for working with container registries, known formally as the [OCI Distribution Specification](https://github.com/opencontainers/distribution-spec/blob/main/spec.md). Beginning in Helm 3, you can use container registries with OCI support to store and share Helm chart packages. From Helm v3.8.0, OCI support is enabled by default.
```

Comment:
Perhaps too much detail on the helm versions?

```suggestion
  - `OCI Registry`: The [Open Container Initiative](https://opencontainers.org/) (OCI) defines the specifications and standards for container technologies. This includes the API for working with container registries, known formally as the [OCI Distribution Specification](https://github.com/opencontainers/distribution-spec/blob/main/spec.md). Beginning in Helm 3, you can use container registries with OCI support to store and share Helm chart packages.
```

Reply:
+1